### ISSUE TITLE
nothing happens

### ISSUE THREAD

**Issue by truesamurai** [Marten4n6/EvilOSX#96]
When I run a module in EvilOSX then i get a message it is added to the qeue of target IP.
But nothing happens then. I waited but nothing happened.
With webcam module I get response: Module server error / Incorrect padding

**Comment by Highpressure** [comment]
its because of the wrong path. you need to change it to e.g. /var/tmp as the current directory in the module has no write permission unless you are root - see my raised issue for details

**Comment by sh1va73** [comment]
Thanks! I tried your fork but now I get: connection refused.

**Comment by truesamurai** [comment]
Thanks, now the error is gone. 
It saves an image now, but when I try to open the image then I get an error: Can not open the image because it contains errors.

**Comment by Highpressure** [comment]
as I just changed the path where the modules are writing / caching stuff, this error is not related to my updates.
are you local or via internet connected and do you use virtualization or real machines?
try following: open a reverseshell (e.g. eggshell from github) and build up a connection from your macos. this way you will see if you have correct NAT, Firewall and Routing in place.

**Comment by Highpressure** [comment]
very strange. 
the image is still created with the screencapture command on the osx and then transfered the way it was ment by the script - I do not edit this. is the machine on your network or the internet and is it virtual? Only thing I could think of, is that somewhere packet inspection messes up with the transfer.. but very unlikely. What size is it? 0 bytes or does something reach your end?

**Comment by truesamurai** [comment]
Yes both machines are on my own network, I do not use VirtualBox. It was few kb. I will do some more testing again.
